packages/dropdowns/demo/combobox.stories.mdx

```diff
@@ -50,6 +50,15 @@ import { COMBOBOX_ITEMS as ITEMS, DROPDOWN_PLACEMENT as PLACEMENT } from './stor
       isLabelHidden: { name: 'hidden', table: { category: 'Label' } },
       hint: { name: 'children', table: { category: 'Hint' } },
       message: { name: 'children', table: { category: 'Message' } },
+      validationLabel: {
+        control: { type: 'text' },
+        table: { category: 'Message' }
+      },
+      validation: {
+        options: ['success', 'warning', 'error'],
+        control: { type: 'radio' },
+        table: { category: 'Message' }
+      },
```

Comment:
This should be removed. The `Combobox` message validation already responds to the top-level component `validation` prop.